tests/unittests/OperatorTest.cpp

```diff
@@ -2139,13 +2139,24 @@ TEST_P(InterpAndCPU, NonSquarePaddingConvolution) {
       IH1.at({0, i, j, 0}) = i * 4 + j - 2 + 1;
     }
 
-  CN = F_->createConv("Conv1", input1, filter, zeroBias, outTy, 2, 1,
+  auto filter1 =
+      mod_.createVariable(ElemKind::FloatTy, {2, 2, 2, 1}, "filter1");
+  auto FH1 = filter1->getHandle();
+  for (size_t i = 0; i < 2 * 2 * 2; i++) {
+    FH1.raw(i) = pow(2.0, i);
+  }
+  auto *zeroBias1 = mod_.createVariable(ElemKind::FloatTy, {2}, "bias1");
+  zeroBias1->getPayload().zero();
+
+  CN = F_->createConv("Conv1", input1, filter1, zeroBias1, outTy, 2, 1,
```

Comment:
You probably want to use a different function for it. F_ was already used for the first part of the test.